src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,17 @@
+.calendar {
+  width: 706px;
```

Comment:
Do not hardcode width, you could calculate it. For this purposes you can move `gap` and `padding` to variables.

This way you could change some size(gap, padding, column width) and  calendar width recalculating would be automatic.

For better understanding you could rewatch  [this video](https://mate.academy/learn/css-advanced/sass#/video/924) from 9:10 timing(or from the start if you want).